src/components/EmptyCart.jsx

```diff
@@ -0,0 +1,31 @@
+import { useNavigate } from "react-router-dom";
+import { Flex, Heading, Text, Button } from "@sparrowengg/twigs-react";
+const EmptyCart = () => {
+  const navigate = useNavigate();
+  const handleNavigate = () => navigate("/home");
+  return (
+    <>
+      <Flex
+        flexDirection="column"
+        justifyContent="space-evenly"
+        alignItems="center"
+        css={{ height: "100%" }}
+      >
+        <Heading css={{ textAlign: "center" }}>
+          Your cart is currently empty!
+        </Heading>
+        <img src={"./empty-cart.png"} alt="" />
```

Comment:
@arun-121 can you try to add alt text what if the img breaks, it may show the text right?

Reply:
added alt text